nerfstudio/model_components/losses.py

```diff
@@ -226,7 +227,7 @@ def ds_nerf_depth_loss(
         Depth loss scalar.
     """
     loss = -torch.log(weights + EPS) * torch.exp(-((steps - termination_depth[:, None]) ** 2) / (2 * sigma)) * lengths
-    return torch.mean(loss.sum(-2))
+    return torch.mean((loss * depth_mask[:, None]).sum(-2))
```

Comment:
Can you avoid editing the function signature of this loss function? That makes the change more minimal.
```
loss = loss * (termination_depth > 0)[:, None]
```